_posts/2019-11-13-Upgrading-a-Fairly-Large-Rails-Application-from-4.2-to-5.2.markdown

```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+title: "Upgrading a Fairly Large Rails Application from 4.2 to 5.2"
+date: 2019-11-13 12:30:00
+categories: ["rails", "upgrades", "case-study"]
+author: "abizzinotto"
+---
+
+We recently collaborated with [Amitree](https://www.home.amitree.com/) on a [Rails upgrade](https://fastruby.io) for their application which "is an AI assistant for email, integrated deeply within Google and Microsoft’s products and is in heavy use by real estate professionals" and were able to speak to them about their experience with [Ombu Labs](https://www.ombulabs.com).
```

Comment:
I believe that every time you say Ombu Labs you should say Fast Ruby instead. 

@etagwerker what do you think?

Reply:
@ABizzinotto @lubc Yes, I think that is a good idea.

Reply:
@etagwerker @lubc my only concern is this could cause concerns with Amitree so we'd need to run the article by then again? I reviewed the feedback they sent and the whole conversation revolved around the Ombu Labs brand. What do you think?

Reply:
@ABizzinotto then leave it as is

Reply:
Ok. @lubc I'll resolve this conversation then.